proxy/envoy/config.go

```diff
@@ -167,8 +167,15 @@ func buildHTTPListener(routeConfig *HTTPRouteConfig, ip string, port int, rds bo
 		config.RouteConfig = routeConfig
 	}
 
+	var sslContext *SSLContext
+	// Build the sslContext for http inbound listener (only inbound listener does not use rds)
+	if !rds && mesh.EnableAuth {
```

Comment:
We need a way to differentiate the inbound vs outbound traffic. How do we differentiate them if not by RDS?